test/Microsoft.Azure.WebJobs.Host.UnitTests/Listeners/ListenerFactoryTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Indexers;
+using Microsoft.Azure.WebJobs.Host.Listeners;
+using Microsoft.Azure.WebJobs.Host.Protocols;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Microsoft.Azure.WebJobs.Host.Triggers;
+using Moq;
+using Xunit;
+using System.Collections.ObjectModel;
+
+namespace Microsoft.Azure.WebJobs.Host.UnitTests.Listeners
+{
+    public class ListenerFactoryTests
+    {
+        CancellationToken ct = default(CancellationToken);
+
+        async Task<IListener> GetListener(TraceWriter trace)
```

Comment:
Out of curiosity, any particular reasoning? Agree with prod code, for tests I don't use as much care as I maybe should. Speed up test indexing perhaps?